kornia/geometry/transform/crop/crop2d_contrib.py

```diff
@@ -0,0 +1,182 @@
+from typing import Optional
+
+import kornia
+import torch
+from .crop2d import crop_by_boxes as sampling_by_boxes, validate_bboxes
+
+
+def generate_dst_bbox_from_bbox(boxes: torch.Tensor):
```

Comment:
As I mentioned I think this can be one of the first functions of `kornia.geometry.bbox`.

In this particular case I'd suggest to rename `generate_bbox_from_points` since from my understanding this function takes as input four 2D points and creates a bounding box representation [x, y, h, w]. Am I right ? Besides this needs some asserts and tests.